build-system/tasks/e2e/index.js

```diff
@@ -76,14 +79,29 @@ async function e2e() {
   // create mocha instance
   require('@babel/register');
   require('./helper');
-  const mocha = new Mocha();
 
-  // add test files to mocha
-  config.e2eTestPaths.forEach(path => {
-    glob.sync(path).forEach(file => {
+  const mocha = new Mocha({
+    reporter: argv.testnames ? '' : ciReporter,
```

Comment:
Very cool! Does `argv.testnames` not need a reporter? For the unit tests, here's the one we use:

https://github.com/ampproject/amphtml/blob/7b541fac90d8757d5776293d84bdcc98d465f96f/build-system/tasks/runtime-test/index.js#L250-L252

Reply:
If a reporter isn't specified, Mocha uses the default `spec` reporter. I think it's the same as the unit test reporter - looks the same!